create-customer/src/test/java/no/unit/nva/customer/create/CreateCustomerHandlerTest.java

```diff
@@ -42,32 +46,33 @@
     public void setUp() {
         customerServiceMock = mock(CustomerService.class);
         environmentMock = mock(Environment.class);
+        customerMapper = new CustomerMapper(SAMPLE_NAMESPACE);
         when(environmentMock.readEnv(ALLOWED_ORIGIN_ENV)).thenReturn(WILDCARD);
-        handler = new CreateCustomerHandler(customerServiceMock, environmentMock);
+        handler = new CreateCustomerHandler(customerServiceMock, customerMapper, environmentMock);
         outputStream = new ByteArrayOutputStream();
         context = Mockito.mock(Context.class);
     }
 
     @Test
     @SuppressWarnings("unchecked")
     public void requestToHandlerReturnsCustomerCreated() throws Exception {
-        Customer customer = new Customer.Builder()
+        CustomerDb customerDb = new CustomerDb.Builder()
                 .withName("New Customer")
                 .build();
-        when(customerServiceMock.createCustomer(customer)).thenReturn(customer);
+        when(customerServiceMock.createCustomer(customerDb)).thenReturn(customerDb);
```

Comment:
Do you need a mock here? I prefer no mocks if we have test setup that allows us to avoid mocking. Mocking in tests are like writing expectations for test code, and not actually testing implementation. 

I know in some cases this is required, but AFAIK we already have a local dynamodb available during tests.

Reply:
The persistence is tested in the commons module, but I see your point. We can look into updating tests in the future.